package.json

```diff
@@ -41,28 +41,28 @@
     "@commitlint/cli": "^9.0.1",
     "@commitlint/config-conventional": "^9.0.1",
     "@types/jest": "^26.0.7",
```

Comment:
do we need the jest types? I think jest has definitions included by now. If we do keep it, we should update to match the jest version

Reply:
I will try to remove them

Reply:
0b05f5b14c9f4ecf653c11d23ace6c52a7d6fe00